app/src/main/java/nu/yona/app/api/manager/impl/NotificationManagerImpl.java

```diff
@@ -161,6 +162,87 @@ public void onError(Object errorMessage) {
+                                        } else {
+                                            message.setNotificationMessageEnum(NotificationMessageEnum.getNotificationMessageEnum(message.getType(), message.getChange()));
+                                        }
+                                        if (message.getLinks() != null && message.getLinks().getYonaUser() != null
+                                                && !TextUtils.isEmpty(message.getLinks().getYonaUser().getHref())) {
+                                            if (message.getEmbedded() == null) {
+                                                message.setEmbedded(new Embedded());
+                                            }
+                                            message.getEmbedded().setYonaUser(getYonaUser(message.getLinks().getYonaUser().getHref()));
+                                        }
+                                        String uploadDate = "";
+
+                                        String createdTime = message.getCreationTime();
+                                        try {
+                                            Date date = sdf.parse(createdTime);
+
+                                            Calendar futureCalendar = Calendar.getInstance();
+                                            futureCalendar.setTime(date);
+
+                                            uploadDate = DateUtility.getRelativeDate(futureCalendar);
+                                            message.setStickyTitle(uploadDate);
+                                        } catch (Exception e) {
+                                            AppUtils.throwException(ActivityManagerImpl.class.getSimpleName(), e, Thread.currentThread(), null);
+                                        }
+                                        resultYonaMessagesObj.setEmbedded(embedded);
+                                        if (!isProcessUpdate && message != null && message.getLinks() != null && message.getLinks().getYonaPreocess() != null
+                                                && !TextUtils.isEmpty(message.getLinks().getYonaPreocess().getHref())) {
+                                            isAnyProcessed = true;
+                                            MessageBody body = new MessageBody();
+                                            body.setProperties(new Properties());
+                                            postMessageForProcess(message.getLinks().getYonaPreocess().getHref(), body);
+                                        }
+                                    }
+                                    if (isAnyProcessed) {
+                                        getMessageWithUrl(urlForMessagesFetch, isUnreadStatus, listener, true);
+                                    }
+                                    APIManager.getInstance().getAuthenticateManager().getUserFromServer();
```

Comment:
Given that you're not using the return value, why are you fetching the user from the server here?

Reply:
i will update this in next commit

Reply:
OK